### DISCUSSION TITLE
How to edit Comment column from RESX files?

### DISCUSSION THREAD

**Discussion by maykQics** [WeblateOrg/weblate#12350] [Q&A] [answered]
Hi, I would like to ask, whether it is possible in Weblate edit Comment from RESX files? We use this column in client resource string automatic generation. I can see that values as "Source string description" for every translation string, but it is not editable there. Is it possible to achieve that somehow in weblate?

**Comment by nijel** [comment, marked as answer]
No, this field is not editable within Weblate. You can set an *explanation* for each string, but that is kept in Weblate only.
